static/js/pages/instances/actions/StartStopInstanceBtn.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { FC, useState } from "react";
+import { LxdInstance } from "types/instance";
+import { NotificationHelper } from "types/notification";
+import { useQueryClient } from "@tanstack/react-query";
+import { startInstance, stopInstance } from "api/instances";
+import { queryKeys } from "util/queryKeys";
+import ConfirmationButton from "components/ConfirmationButton";
+
+interface Props {
+  instance: LxdInstance;
+  notify: NotificationHelper;
+  hasCaption?: boolean;
+}
+
+const StartStopInstanceBtn: FC<Props> = ({
+  instance,
+  notify,
+  hasCaption = true,
+}) => {
+  const [isLoading, setLoading] = useState(false);
+  const queryClient = useQueryClient();
+
+  const isRunning = instance.status === "Running";
+  const isStopped = instance.status === "Stopped";
```

Comment:
What do you mean by "fails"? I don't think we should consider every other status (error, unknown) as "stopped", otherwise this means that you would be able to start an errored/unknown instance. Is this the correct behaviour? If yes, then I will be happy to change as you suggest. Otherwise, as it is now, we have 3 cases:

1) Instance running: show the "Stop" button (which stops the instance if clicked)
2) Instance stopped: show the "Start" button (which starts the instance if clicked)
3) Other status: _component is not rendered as we would not be able to stop nor start it_

Or did I misunderstand anything about the errored/unknown statuses?

Reply:
Hmm, good question. I was assuming you would still want to try to start an instance from an error state.

Reply:
Can you ask the core team in the LXD UI channel if we can start from an error state? Or generally which other states than running/stopped are actually in use and how the start/stop should behave in them.